pkgs/tools/misc/ncdu/default.nix

```diff
@@ -19,6 +19,8 @@ stdenv.mkDerivation rec {
 
   PREFIX = placeholder "out";
 
+  ZIG_FLAGS = "-Drelease-safe -Dcpu=baseline"; # see https://github.com/NixOS/nixpkgs/issues/169461
```

Comment:
Please also add a short comment why we do this in addition to the link.

Reply:
Done